examples/kind-example/pom.xml

```diff
@@ -0,0 +1,106 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>io.dekorate</groupId>
+  <artifactId>kind-example</artifactId>
+  <version>3.1-SNAPSHOT</version>
+  <name>Dekorate :: Examples :: Kind on Kubernetes</name>
```

Comment:
I would name it `Kind cluster` or something like this, instead of Kind on Kubernetes because Kind is already Kubernetes (in docker)

Reply:
very good point